LeaderboardBackend/Models/Entities/NumericalMetric.cs

```diff
@@ -0,0 +1,96 @@
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace LeaderboardBackend.Models.Entities;
+
+/// <summary>
+///     Represents a `NumericalMetric` for a `Category`.
+/// </summary>
+public class NumericalMetric : BaseEntity
+{
+	/// <summary>
+	///     The unique identifier of the `NumericalMetric`.<br/>
+	///     Generated on creation.
+	/// </summary>
+	public long Id { get; set; }
+
+	/// <summary>
+	///     The display name of the `NumericalMetric`.
+	/// </summary>
+	/// <example>Foo</example>
+	[NotNull]
+	[Required]
+	public string? Name { get; set; }
+
+	/// <summary>
+	///     The minimum value of the `NumericalMetric`. The default is 0.
+	/// </summary>
+	[Required]
+	public long Min { get; set; }
+
+	/// <summary>
+	///     The maximum value of the `NumericalMetric`. The default is `Min` + 1.
+	/// </summary>
+	[Required]
+	public long Max { get; set; }
```

Comment:
This should probably default to the integer limit. Otherwise, there are literally only two values it can even be by default.

Reply:
I'll swap to `int`s over `long`s, sure. This will clash with the DB design, which calls for `min` and `max` to be `double`s, heads up.